app/Domains/Users/Policies/UserPolicy.php

```diff
@@ -36,7 +36,7 @@ public function view(User $user): bool
     /**
      * Determine whether the user can create new users in the application or not.
      *
-     * @param  User $user  The resource entity from the authenticated user.
+     * @param User $user The resource entity from the authenticated user.
```

Comment:
Hey Can u align the User class with the same start as the return type in the docblock.